### PR TITLE
Feat/live 8638 LLM Stax Early Security Checks pixel polish

### PR DESCRIPTION
**Pull request by ofreyssinet-ledger**
### 📝 Description

Pixel polish about the LLM Early Security Checks flow for Stax.

Includes changes about:
- error rendering across the entire app:
  - new design (new dimensions of icon box + using new icons)
  - refactoring it to make the basic blocks (`GenericInformationBody`, rendering icon, title, subtitle and description) easily reusable without an error object.
  - adaptation of existing usages to use new icons
- change device language drawer pixel polish
- early security checks design

### ❓ Context

- **Impacted projects**: `llm` 
- **Linked resource(s)**: [LIVE-8638] 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://github.com/LedgerHQ/ledger-live/assets/91890529/9c43d6a5-9cec-46f9-96e7-2be953f0e07a

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-8638]: https://ledgerhq.atlassian.net/browse/LIVE-8638?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ui/packages/native/src/components/Text/getTextStyle.ts [hunk 1]
```diff
@@ -80,7 +80,7 @@ export function getTextTypeStyle({ bracket }: { bracket?: boolean }): Record<
     },
     bodyLineHeight: {
       fontFamily: "Inter",
-      lineHeight: "20px",
+      lineHeight: "21px",
```
Comment (ofreyssinet-ledger) [inline libs/ui/packages/native/src/components/Text/getTextStyle.ts:83]: fix for a warning about missing unit

File: apps/ledger-live-mobile/src/components/GenericErrorView.tsx [hunk 2]
```diff
@@ -17,10 +16,9 @@ type Props = {
   // in such case, the outerError is GenuineCheckFailed and the actual error is still error
   outerError?: Error | null;
   withDescription?: boolean;
-  withIcon?: boolean;
```
Comment (ofreyssinet-ledger) [inline apps/ledger-live-mobile/src/components/GenericErrorView.tsx:20]: unused (always true)

File: apps/ledger-live-mobile/src/components/GenericInformationBody.tsx
Comment (mle-gall) [inline apps/ledger-live-mobile/src/components/GenericInformationBody.tsx:1]: Great addition !

**Review by mle-gall — APPROVED** [review]

File: apps/ledger-live-mobile/src/components/GenericInformationBody.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Flex, Text, BoxedIcon, Icons } from "@ledgerhq/native-ui";
+import { NewIconType } from "@ledgerhq/native-ui/components/Icon/type";
+
+/**
+ * Component rendering an icon, a title, a subtitle and a description.
+ */
+export const GenericInformationBody: React.FC<{
```
Comment (alexandremgo) [inline apps/ledger-live-mobile/src/components/GenericInformationBody.tsx:8]: Nice 🔥

File: apps/ledger-live-mobile/src/components/BleDevicePairingFlow/BleDevicePairing.tsx [hunk 4]
```diff
@@ -147,23 +139,7 @@ const BleDevicePairing = ({ deviceToPair, onPaired, onRetry }: BleDevicePairingP
...
-              iconColor="primary.c80"
-            />
-          </Flex>
-          <Text mt={4} mb={2} textAlign="center" variant="h4" fontWeight="semiBold">
-            {title}
-          </Text>
-          <Text variant="body" mb={8} color="neutral.c80" textAlign="center">
-            {subtitle}
-          </Text>
+          <GenericInformationBody title={title} description={subtitle} />
```
Comment (alexandremgo) [inline apps/ledger-live-mobile/src/components/BleDevicePairingFlow/BleDevicePairing.tsx:142]: QA should check the pairing flow just to be 100% sure we're good

**Review by alexandremgo — APPROVED** [review]

File: apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx [hunk 5]
```diff
@@ -635,13 +634,6 @@ export function renderError({
   iconColor?: string;
   device?: Device;
   hasExportLogButton?: boolean;
-  /*
-   * Used when rendering a Bluetooth disabled error
-   *
-   * If "drawer", the component will be rendered as a content to be rendered in a drawer.
-   * If "view", the component will be rendered as a view. Defaults to "view".
-   */
-  renderedInType?: "drawer" | "view";
```
Comment (ofreyssinet-ledger) [inline apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx:644]: this has been removed, it's used only in `BluetoothDisabled` and it was replaced by a more practical `IsInDrawerContext`

File: apps/ledger-live-mobile/src/components/RequiresBLE/BluetoothDisabled.tsx [hunk 6]
```diff
@@ -17,15 +17,10 @@ export type Props = {
...
- *   If "view", the component will be rendered as a view. Defaults to "view".
  */
-const BluetoothDisabled: React.FC<Props> = ({
-  onRetry,
-  forceOpenSettings = false,
-  componentType = "view",
-}) => {
+const BluetoothDisabled: React.FC<Props> = ({ onRetry, forceOpenSettings = false }) => {
   const { t } = useTranslation();
+  const { isInDrawer } = useContext(IsInDrawerContext);
```
Comment (alexandremgo) [inline apps/ledger-live-mobile/src/components/RequiresBLE/BluetoothDisabled.tsx:23]: Ok way better ;)

Comment (ofreyssinet-ledger) [reply apps/ledger-live-mobile/src/components/RequiresBLE/BluetoothDisabled.tsx:23]: I love Context for stuff like that ❤️

File: apps/ledger-live-mobile/src/context/IsInDrawerContext.ts [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const IsInDrawerContext = React.createContext<{ isInDrawer?: boolean }>({ isInDrawer: false });
```
Comment (alexandremgo) [inline apps/ledger-live-mobile/src/context/IsInDrawerContext.ts]: Nice 👌

**Review by alexandremgo — APPROVED** [review]
Clean changes 🦖

**Pull request merged**